### ISSUE TITLE
Filter active vs inactive

### ISSUE THREAD

**Issue by samoehlert** [esnet-security/SCRAM#30]
It would be nice to be able to filter so i can quickly tell how many entries we have active in two places

1. in the admin panel -> route_manager -> entries
2. in the webUI, the number on the homepage should just be active blocks i think

**Comment by grigorescu** [comment]
Good idea. Could just add a [pill badge](https://getbootstrap.com/docs/4.0/components/badge/#pill-badges) to the navbar with active/total, since that seems like a good thing to just display prominently and won't take much space.

**Comment by samoehlert** [comment]
The admin panel was already taken care in #108
